src/utils/sortFilterTargets.ts

```diff
@@ -0,0 +1,60 @@
+import { TTarget } from "@types";
+
+const sourceMapping = {
+  strategy: "executive",
+  plan: "executive",
+  law: "legislative",
+  framework: "legislative",
+  ndc: "executive",
+  policy: "executive",
+};
+
+export const sortFilterTargets = (targets: TTarget[]) => {
+  let t: TTarget[] = [];
+  if (!targets) return t;
+  t = targets.filter((target) => target["Visibility status"] === "published");
+
+  const netZeroKey = "Net zero target?";
+  const ghgKey = "GHG target";
+  const sourceKey = "Source";
+  const sectorKey = "Sector";
+  const sectorToInclude = "economy-wide";
+
+  /* 
+    Net zero targets should be first, 
+    then GHG targets, 
+    then legislative (legal) targets,
+    then targets in the economy-wide sector
+  */
+  t = t.sort((a, b) => {
+    if (a[netZeroKey] === "TRUE" && b[netZeroKey] === "FALSE") {
+      return -1;
```

Comment:
Again just style we could ease reading with:
```
const a_after_b = 1;
const a_before_b = -1;
```

Reply:
It is fairly standard sort syntax but I'm happy to make the change

Reply:
Its not a request to make a change mearly a suggestion